<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>29</width>
    <height>29</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="srvIPLineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="srvPortLineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="tcpSendText">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>40</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="tcpRecvText">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="tcpEstablishButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tcpSendButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>160</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="relayCheckBox_1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>440</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>继电器</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="relayCheckBox_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>440</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>继电器</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="relayCheckBox_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>440</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>继电器</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="relayCheckBox_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>440</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>继电器</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="relayCheckBox_5">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>440</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>继电器</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="relayCheckBox_6">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>440</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>继电器</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="relayCheckBox_7">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>440</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>继电器</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="relayCheckBox_0">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>继电器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="runSystemButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>启动系统</string>
    </property>
   </widget>
   <widget class="QPushButton" name="relayButton_1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>390</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>闭合</string>
    </property>
   </widget>
   <widget class="QPushButton" name="relayButton_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>390</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>闭合</string>
    </property>
   </widget>
   <widget class="QPushButton" name="relayButton_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>390</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>闭合</string>
    </property>
   </widget>
   <widget class="QPushButton" name="relayButton_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>390</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>闭合</string>
    </property>
   </widget>
   <widget class="QPushButton" name="relayButton_5">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>390</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>闭合</string>
    </property>
   </widget>
   <widget class="QPushButton" name="relayButton_6">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>390</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>闭合</string>
    </property>
   </widget>
   <widget class="QPushButton" name="relayButton_7">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>390</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>闭合</string>
    </property>
   </widget>
   <widget class="QPushButton" name="relayButton_0">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>闭合</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>发送区</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>接收区</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setAllRelayButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>430</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>控制所有继电器</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rtcLineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="sysTimeLabel">
    <property name="geometry">
     <rect>
      <x>661</x>
      <y>10</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>sysTime</string>
    </property>
   </widget>
   <widget class="QLabel" name="stateLabel_0">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1号: 状态</string>
    </property>
   </widget>
   <widget class="QLabel" name="stateLabel_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>2号: 状态</string>
    </property>
   </widget>
   <widget class="QLabel" name="stateLabel_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>3号: 状态</string>
    </property>
   </widget>
   <widget class="QLabel" name="stateLabel_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>4号: 状态</string>
    </property>
   </widget>
   <widget class="QLabel" name="stateLabel_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>5号: 状态</string>
    </property>
   </widget>
   <widget class="QLabel" name="stateLabel_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>6号: 状态</string>
    </property>
   </widget>
   <widget class="QLabel" name="stateLabel_6">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>290</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>7号: 状态</string>
    </property>
   </widget>
   <widget class="QLabel" name="stateLabel_7">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>290</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>8号: 状态</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>839</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
